addons/gamification/models/gamification_karma_rank.py

```diff
@@ -35,7 +35,7 @@ def _compute_rank_users_count(self):
     @api.model_create_multi
     def create(self, values_list):
         res = super(KarmaRank, self).create(values_list)
-        users = self.env['res.users'].sudo().search([('karma', '>', 0)])
+        users = self.env['res.users'].sudo().search([('karma', '>', min(res.karma_min))])
```

Comment:
use "mapped" so you are sure to always have a list :) (min(5) will crash)
```suggestion
        users = self.env['res.users'].sudo().search([('karma', '>', min(res.mapped('karma_min')))])
```

Reply:
Done!